backend/app/agents/devrel/github/services/github_mcp_client.py

```
@@ -0,0 +1,69 @@
+            logger.error(f"Unexpected error: {e}")
+            return {"error": f"Unexpected error: {str(e)}"}
+    
+    async def is_server_available(self) -> bool:
+        
+        if not self.session:
+            return False
+        
+        try:
+            async with self.session.get(f"{self.mcp_server_url}/health", timeout=5) as response:
+                return response.status == 200
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)